src/redis.go

```diff
@@ -30,6 +30,8 @@ type argumentList struct {
 	RenamedCommands  sdkArgs.JSON `default:"" help:"Map of default redis commands to their renamed form, if rename-command config has been used in the redis server."`
 	ConfigInventory  bool         `default:"true" help:"Provides CONFIG inventory information. Set it to 'false' in environments where the Redis CONFIG command is prohibited (e.g. AWS ElastiCache)"`
 	ShowVersion      bool         `default:"false" help:"Print build information and exit"`
+	UseTLS           bool         `default:"false" help:"Use TLS when communicating with the Redis server."`
+	SkipTLSVerify    bool         `default:"false" help:"Disable server name verification when connecting over TLS"`
```

Comment:
Is this how we want to name these parameters? I've seem quite variability from integration on how these are named, and I think we could be more uniform here:

https://github.com/newrelic/nri-mysql/blob/9ba52b375cd81319c230ce5d12e22e168c1fcc21/src/mysql.go#L35-L37

https://github.com/newrelic/nri-mssql/blob/a67ac6465a2ea5882c762ca143d10529775b0f27/src/args/argument_list.go#L20-L22

https://github.com/newrelic/nri-couchbase/blob/c8202e88767bce3848751650d2509489841575d4/src/arguments/arguments.go#L18-L21

https://github.com/newrelic/nri-elasticsearch/blob/542ba5dae1fd19efe7f7256e4fad646ea771e1f0/src/elasticsearch.go#L23-L26

https://github.com/newrelic/nri-consul/blob/1bb3eb0b309c1db6250c8f357ce95425078f56db/src/args/argument_list.go#L18-L21

https://github.com/newrelic/nri-mongodb/blob/14d78430b1e120d0e4f2c32df428ac1db1f5f191/src/arguments/arguments.go#L23-L27

https://github.com/newrelic/nri-postgresql/blob/c81430f45e82a2591946f943508659487240dfbf/src/args/argument_list.go#L20-L22

https://github.com/newrelic/nri-rabbitmq/blob/c7634e35f12607cd16027be29b05779cb02393ca/src/args/args.go#L17

Reply:
I had the same doubt, so far we have been not following any standard for that. Most of the repos use different namings
UseSSL, SSL, EnableSSL, EnableTLS

Regarding skip certificate validation is even changing more.

That is the very same naming of redis library, I am fine with any of those

Reply:
UseSSL seems to be the most common one together with TrustServerCertificate

Reply:
Nope, I'm also against using `SSL` on things that are `TLS`. My favourite would be `-tls`, but... 
![xkcd: Standards](https://imgs.xkcd.com/comics/standards.png)

Since `UseSSL` is the most common, let's go for `UseTLS`.

WDYT about `s/SkipTLSVerify/TLSInsecureSkipVerify`?

Reply:
done!